index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!-- <h2>Hello World</h2> -->
+
+    <main class="main-wrapper">
+      <div class="header">
+        <h1 id="title">
+          Survey Form
+        </h1>
+        <p id="description">
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptate quasi molestias sunt, ullam placeat qui commodi libero porro dignissimos blanditiis et animi doloremque rem, ipsam numquam omnis deserunt, quos iste.
+        </p>
+      </div>
+
+      <section class="form-section">
+        <form action="" id="survey-form">
+          <div class="form__group">
+            <input type="text" class="form__group-input" id="name" placeholder="Name">
+            <label for="name" id="name-label" class="form__group-label">Name</label>
+          </div>
```

Comment:
I think it will be better to replace the `section` tag with a `div`. Following the HTML semantics rules, a section should have a header element too.